static/lib/tutorials/en_US/expresstohapi.md

```diff
@@ -303,6 +285,24 @@ const getDate = {
     }
 };
 ```
+To get access to the options in hapi, you simply refer to the `options` object when you create the plugin:
```

Comment:
I think we move things around like that, this text and the one here: https://github.com/hapijs/hapi.dev/pull/540/files#diff-b93eeadd658d6cd7b04c2a0be3eddbbe9d02fbd17272a8965382cbd771616cdaR253 don't match that much anymore IMO and we should rephrase them.

Reply:
1. I agree about the rephrase but only this part is problematic: "In hapi, you set the options when you register the plugin. Lets have a look:".
2. the move of the examples is necessary for logical understanding of similar capabilities of express and hapi.
anyway I will push new version with correct explanation .
thank you :)

Reply:
I pushed a new version.